trino-connector/integration-test/src/test/resources/trino-ci-testset/testsets/hive/00011_row.sql

```diff
@@ -68,23 +68,23 @@ FROM (
 
 INSERT INTO source_tb1 VALUES (1, 'Alice'), (1, 'Alice'), (2, 'Bob');
 
-INSERT INTO test_row_basic SELECT ROW(id, name) FROM source_tb1;
+INSERT INTO tb01 SELECT ROW(id, name) FROM source_tb1;
 
-SELECT person.id, COUNT(*) FROM test_row_basic GROUP BY person.id ORDER BY person.id;
+SELECT person.id, COUNT(*) FROM tb01 GROUP BY person.id ORDER BY person.id;
 
-DROP TABLE gt_hive.gt_row_db1.test_row_basic;
+DROP TABLE gt_hive.gt_row_db1.tb01;
 
 DROP TABLE gt_hive.gt_row_db1.source_tb1;
 
-DROP TABLE gt_hive.gt_row_db1.test_nested_row;
+DROP TABLE gt_hive.gt_row_db1.tb02;
 
-DROP TABLE gt_hive.gt_row_db1.test_mixed_row;
+DROP TABLE gt_hive.gt_row_db1.tb03;
 
 DROP TABLE gt_hive.gt_row_db1.source_tb2;
 
 DROP TABLE gt_hive.gt_row_db1.source_tb3;
 
-DROP TABLE gt_hive.gt_row_db1.test_row_in_array_map;
+DROP TABLE gt_hive.gt_row_db1.tb04;
 
```

Comment:
The original name is Ok. Why do you change it?

Reply:
Align these table name with the old cases.